src/tutorials/narratives-how-to-write.rst

```diff
@@ -0,0 +1,345 @@
+We can then drag the markdown file onto the Narrative Debugger page and you should see something like the top half of this figure:
+
+..
+  TODO- update links above once debugger functionality in nextstrain.org
+
+.. image :: ../images/narratives_debugger_screenshot.jpg
+   :alt: Screenshot of the example narrative loaded in the narratives debugger and a preview of the opening slide
+
+
+We can see the titles of the three slides & their associated datasets (see the following section for details on how titles are defined), and hovering over these should show you the full appearance of the slide.
+To the right of the titles are their associate datasets, each with a series of icons next to them representing the main + sidecar JSONs associated with this dataset.
+The icons represent whether the dataset exists on nextstrain.org -- in this case, they are all green (success) or grey (not attempted).
+When debugging a narrative it's easy to make syntax errors resulting in datasets which don't load -- you could try changing the URLs in the markdown file and dragging the file back onto the debugger to see this.
+Clicking on the title of a slide will jump into the narrative at that slide, or you can click the "View Narrative" button to start from the beginning.
+
+
+.. note::
+
+  The narrative file you drag onto the debugger remains within your browser and is not sent to the nextstrain server.
+  The datasets it specifies must be accessible through nextstrain.org and are fetched (downloaded) when you are testing a narrative; these can include private Nextstrain Groups assuming you are logged into nextstrain.org & can access those private datasets.
+
+
+
+A typical writing process
+=========================
+
+Hopefully the above section gives you enough to get started writing your own narratives.
+There are more technical details to follow, but if you want to get started we encourage trying it out!
+There are plenty of ways to approach the task, but we find the following workflow tends to work well:
+
+
+.. graphviz::
+    :align: center
+
+    strict digraph {
+        rankdir="LR";
+        auspice [shape="tab" style="filled" fillcolor="#c7e9b4" label="nextstrain.org/...\nto choose desired \nview of data"]
+        md [shape="note" style="filled" fillcolor="#41b6c4" label="Narrative file\nwe are writing\n(markdown)"]
+        debugger [shape="tab" style="filled" fontcolor="white" fillcolor="#225ea8" label="Narratives debugger\nto test narrative\nas we go"]
+        
+        auspice -> md [label="copy\nURL" fontcolor="#7fcdbb" fillcolor="#7fcdbb" color="#7fcdbb"]
+        md -> auspice [label="repeat" fontcolor="#7fcdbb" fillcolor="#7fcdbb" color="#7fcdbb" splines=curved]
+        md -> debugger [label="drag &\ndrop" fontcolor="#1d91c0" fillcolor="#1d91c0" color="#1d91c0"]
+        debugger -> md [label="repeat" fontcolor="#1d91c0" fillcolor="#1d91c0" color="#1d91c0" splines=curved]
+    }
```

Comment:
I think would be nice to use a sans-serif font here, like the surrounding body text and our other graphviz diagrams.

Reply:
Done - https://github.com/nextstrain/docs.nextstrain.org/compare/03a1276b5dfebbb8d81a3447d64cd31239b9f4e4..477e27d73e3cde37e40c662e0b3aea0a9c4cb987